components/slider/README.md

```diff
@@ -47,4 +47,39 @@ modules from the node_modules folder.
 ~~~~
 
 To overwrite the default styles, simply create new rules for the class names that
-you wish to change and include them after the default styles.
\ No newline at end of file
+you wish to change and include them after the default styles.
+
+## Updating the slider's state
+
+You are able to update dynamically the sider state for the following properties:
+
+* orientation - How the slider is oriented. Either 'horizontal' or 'vertical'.
+* step - The scroll step of the slider that is used when scrolling with the mouse. Should be a number. Non positive values are also accepted and they can be used to invert the scroll direction.
+
+You can set them via `setAttribute` or directly using the setters like `slider.orientation = 'horizontal'`. 
+
+For example:
+```javascript
+const slider = document.querySelector('gameface-slider');
+
+slider.orientation = 'horizontal'; // or 'vertical'. If is set something different the slider orientatil will fallback to default one - 'vertical'
+// Or
+slider.setAttribute('orientation', 'horizontal');
+
+slider.step = 20; // If you want to invert the mouse wheel you can set negative value. For example -20.
+// Or
+slider.setAttribute('step', '20');
+```
+
+You can get the current orientation or step in the following way:
+
+```javascript
+console.log(slider.orientation);
+console.log(slider.step);
+```
+
+You can also update the current handle position of the slider using the `scrollTo` method. It receives one argument that is the new position of the handle in percents.
+
+```javascript
+slider.scrollTo(50); // Will set the handle to position that is 50% of the available scroll bar area.
```

Comment:
A note only: This is not entirely true, but when Tony finishes with this task: [COH-18800](https://coherent-labs.atlassian.net/browse/COH-18800) it should be

Reply:
Will leave it for now like it is and it will be fixed OoB :D